tests/e2e/lib/pages/wpcom/login.js

```diff
@@ -29,10 +29,11 @@ export default class LoginPage extends Page {
 		await waitAndClick( this.page, continueButtonSelector );
 
 		// sometimes it failing to type the whole password correctly. Trying to wait for the transition to happen
-		this.page.waitFor( 1000 );
+		this.page.waitFor( 2000 );
 		await waitAndType( this.page, passwordSelector, password );
+		this.page.waitFor( 2000 );
 
-		await waitAndType( this.page, passwordSelector, password );
+		// await waitAndType( this.page, passwordSelector, password );
```

Comment:
Do we need to keep this commented out line?

Reply:
addressed in 2fe3811